src/main/java/io/vincenzopalazzo/placeholder/ui/BasicTextFieldPlaceholderUI.java

```diff
@@ -81,7 +86,7 @@ public void installUI(JComponent c) {
     this.disabledBackground = tmp == null ? Color.CYAN : tmp;
     tmp = UIManager.getColor(this.getPrefix() + ".disabledForeground");
     this.disabledForeground = tmp == null ? Color.CYAN : tmp;
-
+    this.textFieldPlaceholder.setBorder(this.border);
```

Comment:
This is my solution to remove the space on the border @zanderson9 but also with the actual implementation, you can change the round border.